grpc-api-client/src/main/java/io/deephaven/grpc_api_client/barrage/chunk/ChunkInputStreamGenerator.java

```diff
@@ -154,18 +152,11 @@ public FieldNodeInfo(final BarrageFieldNode node) {
     }
 
     final class BufferInfo {
```

Comment:
this is just an immutable wrapper for a single long now, right? overkill perhaps?

Reply:
I have yet to do an in-depth review on this code with Ryan. I noticed that a boxed Long is not a bad alternative (also there are other alternatives - like trove). I'm deferring that change here until then.